backends/edk2-compat/include/edk2-svc.h

```diff
@@ -13,7 +13,7 @@
 #define CERT_BUFFER_SIZE        2048
 
 #ifndef SECVARPATH
-#define SECVARPATH "/sys/firmware/secvar/vars"
+#define SECVARPATH "/sys/firmware/secvar/vars/"
```

Comment:
Note: this short commit message is fine, since it's literally a 1 character typo fix that isn't largely consequential.